apps/OboeTester/app/src/main/cpp/InputStreamCallbackAnalyzer.h

```diff
@@ -75,11 +75,11 @@ class InputStreamCallbackAnalyzer : public OboeTesterStreamCallback {
     }
 
 public:
-    PeakDetector            mPeakDetectors[kMaxInputChannels];
+    int32_t mNumChannels = 0;
+    std::unique_ptr<PeakDetector[]> mPeakDetectors;
```

Comment:
There can be cases where getPeakLevel crashes as a certain channel is non-existent or the the unique_ptr is null.

This may be fine as this is a test app.

Reply:
Yeah, it should only be called after setup().
But I should add error checks and return -1.0 on error.